src/@orb.yml

```diff
@@ -1,15 +1,7 @@
 version: 2.1
 
 description: >
-  Sample orb description
-# What will your orb allow users to accomplish?
-# Descriptions should be short, simple, and informative.
-
-# This information will be displayed in the orb registry and is not mandatory.
+  Use ACCELQ Orb to trigger scheduled CI jobs registered in ACCELQ application.
```

Comment:
Consider a more descriptive explanation of the value and benefits of this integration for users who are not yet familiar with ACCELQ. Keywords will help with search results as well. 

What is ACCELQ and how can adding it to my CircleCI Config add value as a user?

Reply:
1. Already have renamed example.yml to execute_accelq_tests.yml
2. Done
3. Integration testing is correct. Its intentionally filled with wrong values. Such that no one triggers the actual job in accelq side and eat up resources. Further the error means agent is not availbale in accelq to run a job. The indication that integration test is working is that the job was successful to login to accelq.